src/blog/2022/11/community-news-10.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Community News October 2022 #10
+subtitle: News from the FlowForge and Node-RED communities
+description: News from the FlowForge and Node-RED communities
+date: 2022-11-04
+authors: ["rob-marcer"]
+---
+
+Welcome to the FlowForge newsletter for October 2022, a montly roundup of what’s happening with both FlowForge and the wider Node-RED community.
+<!--more-->
+If you've got something that you'd like us to share please email [contact@flowforge.com](mailto:contact@flowforge.com).
+
+[**FlowForge 1.0 Released**](http://localhost:8080/blog/2022/10/flowforge-1-released/)   
```

Comment:
Localhost?

Reply:
Ohh, think that's a an issue with how I was running it locally, will fix.

Reply:
I've resolved this.